assignments/lambda-classes.js

```diff
@@ -1 +1,147 @@
 // CODE here for your Lambda Classes
+cl = console.log;
```

Comment:
I love this! Very clever 👍

Reply:
I wish I could take credit for that, but Moises showed us that during his video the other day.